actors/multisig/tests/multisig_actor_test.rs

```diff
@@ -65,3 +70,30 @@ fn test_construction_fail_to_construct_multisig_with_more_than_max_signers() {
     );
     rt.verify();
 }
+
+#[test]
+fn test_happy_path_add_signer() {
+    let msig = Address::new_id(100);
+    let anne = Address::new_id(101);
+    let bob = Address::new_id(102);
+    let chuck = Address::new_id(103);
+    let mut rt = construct_runtime(msig);
+    let initial_signers = vec![anne, bob];
+    let initial_approvals: u64 = 2;
+
+    // construct the multisig actor and add id addrs to runtime
+    let h = util::ActorHarness {};
+    h.construct_and_verify(&mut rt, initial_approvals, 0, 0, initial_signers);
```

Comment:
@jennijuju with this test in place it is now easier to abstract over it and add many test cases as in go